chapters/04-function-development.Rmd

```diff
@@ -10,33 +10,51 @@
   into a bigger more complex functions
 - Process control (if-else, stop, return, switch)
 - Dependency management
+
+TODO: should this be here or later?
+
 - Function documentation (with roxygen2), part 1
 - To use `@examples` to help with creating function
 
-## Final exercise
+## Final exercise {#04-final-exercise}
+
+TODO: INCOMPLETE, need to think about this more.
+
+Complete these tasks as part of the package for your first project assignment.
+The exact functions you should create will depend heavily on what your package
+will do. Get help and guidance from your instructor. Use our examples on the 
+`kenyaweather` package as further guidance.
 
-<!-- NOTE: Preferably something linked to working towards completing the project assignments, maybe to be used as a milestone? -->
+1. Create a new R script using `use_r("..")`
 
 1. Add a `FIXME()` function to your FIXME package
     
     The following code snippet produces ... FIXME.  
+    
     ```{r, eval = FALSE}
     # A line or two of R code that does something useful
     FIXME
     ```
-    Turn it into a function called `FIXME()` that takes arguments for `FIXME` and `FIXME` and include it in your FIXME package.
     
-    Insert a Roxygen Skeleton for this function, making sure to edit the title, description, argument descriptions, and example. Don't forget to run `document()` to produce the help page.
+    Turn it into a function called `FIXME()` that takes arguments for `FIXME`
```

Comment:
Remember to explicitly include process control in at least one of these exercises

Reply:
As mentioned in #70 process control will be referred to with other texts but not in this book.